metron-interface/metron-config/src/app/model/sensor-parser-config.ts

```diff
@@ -23,7 +23,7 @@ export class SensorParserConfig {
   writerClassName: string;
   errorWriterClassName: string;
   invalidWriterClassName: string;
-  parserConfig: {};
+  parserConfig: { [key: string]: any; }
```

Comment:
Typing using `{}` just extends the Object interface. What we really want here is an object literal that's editable by users.